list/LIST_HEADING_README.md

```diff
@@ -0,0 +1,162 @@
+| status                           | fixed value = `current`                                                                                                                                                          |
+| mode                             | fixed value = `snapshot`                                                                                                                                                         |
+| title                            | `CompoundStatement / code/ originalText` or `CompoundStatement / code [@displayName]`                                                                                            |
+| code.coding\[0].system           | fixed value = `http://snomed.info/sct`                                                                                                                                           |
+| code.coding\[0].code             | fixed value = `24781000000107`                                                                                                                                                   |
+| code.coding\[0].display          | fixed value = `Category (EHR)`                                                                                                                                                   |
+| subject                          | this a reference to mapped [Patient](../patient/README.md) from `Topic.subject`                                                                                                  |
+| date                             | `CompoundStatement / AvailabilityTime [@value]` or from `Topic.date`                                                                                                             |
+| orderedBy.coding\[0].system      | fixed value = `http://hl7.org/fhir/list-order`                                                                                                                                   |
+| orderedBy.coding\[0].code        | fixed value = `system`                                                                                                                                                           |
+| orderedBy.coding\[0].display     | fixed value = `Sorted by System`                                                                                                                                                 |
+| encounter                        | this a reference to mapped [Encounter](../encounters/README.md) from `Topic.encounter`                                                                                           |
+| entry[index].item.reference      | Each entry.item is a reference to a resource representing a clinical record entry which has been mapped from the source HL7 - for example, medications, allergies, problems, etc |
+
+- identifier
```

Comment:
Missing `The following List fields are not currently populated by the adaptor:`

Reply:
Corrected